src/controllers/EventController.ts

```diff
@@ -93,7 +93,7 @@ export class EventController {
   @ResponseSchema(AttendanceResponse)
   async signInToEvent(
     @Param('eventID') eventID: number,
-    @Body() appUserRequest: EventSignInRequest
+    @Body() appUserRequest: AppUserEventRequest
   ): Promise<Attendance | undefined> {
```

Comment:
Promise <AttendanceResponse | undefined>

Reply:
I'm happy to merge after you fix this

Reply:
Won't I need a mapper if I do this?

Reply:
yeah - if you look at https://github.com/HKN-UCSD/hkn-server/blob/eventSignInEndpoint/src/controllers/EventController.ts all the entities are mapped to responses explicitly